apps/web/src/server/app/inspector/inspector.controller.js

```diff
@@ -1,10 +1,79 @@
-import { findAppealById } from './inspector.service.js';
+// @ts-check
+
+import * as inspectorSession from './inspector-session.service.js';
+import * as inspectorService from './inspector.service.js';
 
 /** @typedef {import('@pins/inspector').Appeal} Appeal */
+/** @typedef {import('@pins/inspector').AppealOutcome} AppealOutcome */
+/** @typedef {import('@pins/inspector').AppealSummary} AppealSummary */
+/** @typedef {import('@pins/inspector').SiteVisitType} SiteVisitType */
+
+/**
+ * @typedef {Object} ViewDashboardRenderOptions
+ * @property {AppealSummary[]} appeals - A collection of appeals assigned to the user.
+ */
+
+/**
+ * View the dashboard listing the inspector's assigned appeals.
+ *
+ * @type {import('@pins/express').QueryHandler<{}, ViewDashboardRenderOptions>}
+ */
+export async function viewDashboard(_, response) {
```

Comment:
Ideally these should be named in keeping with established conventions in the other controllers, but we do seem to be breaking with that in recent PRs, so I'm not sure it's worth worrying about

Reply:
Yeah, I deliberately deviated. I am using the rails-like convention of

GET
view<Something> – a read only display of something
new<Something>– a route that creates
edit<Something> – a route that edits

Then for post/patch, it's using the verb that the route is performing, like "createBook", "issueDecision" etc

Reply:
That does seem better. Consistency is my main worry, so we need to make sure we update the rest of the codebase to follow the same conventions. But that's beyond the scope of this PR.

Reply:
LGTM and it makes more sense. 👍🏻